tests/terraform/runner/test_runner.py

```diff
@@ -61,6 +61,40 @@ def test_runner_specific_file(self):
         self.assertEqual(summary['failed'], 0)
         self.assertEqual(summary["parsing_errors"], 0)
 
+    def test_check_ids_dont_collide(self):
```

Comment:
i thought we are testing this in tests/terraform/test_scanner_registry.py
it failed builds several times. what is the diff?

Reply:
@schosterbarak I wasn't aware of that test. nice. 
However, that test checked only one registry - this checks across all registries...